src/events/qaMention.js

```diff
@@ -1,9 +1,13 @@
 const { Events } = require('discord.js');
 const { DISCORD_CONFIG } = require('../config');
+const { pollResults } = require('../utils/poll-vote');
 
 module.exports = {
   name: Events.MessageCreate,
   async execute(message) {
+
+    if (message.embeds[0]?.data?.type === 'poll_result') await pollResults(message)  
+
```

Comment:
this can be removed now, since you already created a [pollVotationResults](https://github.com/TeamNovaSoft/discord-bot/pull/29/files#diff-e7608383e0544452f227a7ad2e8dbeb02f7fa564b31c80cfe2138db3256a892aR4-R12) events

Reply:
Complete